nf_core/pipeline-template/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Initial release of {{ name }}, created with the [nf-core](https://nf-co.re/) tem
 
 ### `Fixed`
 
+Clarified conda usage and added an installation tutorial for Singularity since the one on Syllabs' website uses an outdate version of GO Compiler
```

Comment:
```suggestion
- Clarified conda usage and added an installation tutorial for Singularity since the one on Syllabs' website uses an outdate version of GO Compiler
```

Is the https://www.sylabs.io/guides/3.0/user-guide/ URL in the docs no good then? Should we remove / warn people?